project/accounts/factory.py

```diff
@@ -1,19 +1,70 @@
+from turtle import color
 import factory
 from . import models
+from categories.factory import CategoryFactory
 
-class UserFactory(factory.Factory):
+class UserFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = models.User
 
-    first_name = 'John'
-    last_name = 'Doe'
-    admin = False
 
-# Another, different, factory for the same object
-class AdminFactory(factory.Factory):
+class ProfileFactory(factory.django.DjangoModelFactory):
     class Meta:
-        model = models.User
+        model = models.Profile
+
+    user = factory.RelatedFactory(UserFactory, factory_related_name="profile")
```

Comment:
Should this field be called "profile" since it has the related name "profile"?
```suggestion
    profile = factory.RelatedFactory(UserFactory, factory_related_name="profile")
```

Reply:
The field in the profile model is:
user = models.OneToOneField(User, 
on_delete=models.CASCADE, 
related_name="profile")

That was the reason i had it done in that way, i can't find enough resource to help with a OneToOneField using factory_boy plus i am conflicted as to if i have to also include a field in the UserFactory but the model.User only contains a link to the database

Reply:
Also should i change the name of each field which has a related_name to the related_name